packages/@aws-cdk/aws-lambda/lib/scalable-function-attribute.ts

```diff
@@ -0,0 +1,79 @@
+import * as appscaling from '@aws-cdk/aws-applicationautoscaling';
+import { Construct } from '@aws-cdk/core';
+
+/**
+ * Interface for scalable attributes
+ */
+export interface IScalableFunctionAttribute {
+  /**
+   * Scale out or in to keep utilization at a given level. The utilization is tracked by the
+   * LambdaProvisionedConcurrencyUtilization metric, emitted by lambda. See:
+   * https://docs.aws.amazon.com/lambda/latest/dg/monitoring-metrics.html#monitoring-metrics-concurrency
+   *
+   * Allowed values: 0.1 - 0.9.
+   */
+  scaleOnUtilization(props: UtilizationScalingProps): void;
+  /**
+   * Scale out or in based on time.
+   */
+  scaleOnSchedule(id: string, actions: appscaling.ScalingSchedule): void;
+}
+
+/**
+ * Properties for enabling Lambda utilization tracking
+ */
+export interface UtilizationScalingProps extends appscaling.BaseTargetTrackingProps {
+  /**
+   * Target utilization percent for the attribute
+   */
+  readonly targetUtilizationPercent: number;
```

Comment:
Not a Percent anymore, maybe value? In any case should be changed

Reply:
Can I push back on this a bit? I agree that it is not perfect, but I feel like percent is the most apt description for what this metric represents. From the docs:

`ProvisionedConcurrencyUtilization` – For a version or alias, the value of ProvisionedConcurrentExecutions divided by the total amount of provisioned concurrency allocated. For example, .5 indicates that 50 percent of allocated provisioned concurrency is in use.

I argue that this indicates a percentage and the confusion really comes from the metric ranging from .1 to .9 rather than 10 to 90 and not from the property name.

If we keep the name at `targetUtilizationPercent`, users might be confused and use `50` instead of `.5`, which will hit our error handling and be updated to the correct range.

If we change the name to `targetUtilizationValue` or `targetUtilizationAmount`, I think that there will be more confusion about what value or amount to put here.

I've updated all other suggestions, and will update this one separately if you don't agree.

Reply:
I added `Allowed values: 0.1 - 0.9` to the property doc to try and assuage any confusion.

Reply:
> If we keep the name at targetUtilizationPercent, users might be confused and use 50 instead of .5, which will hit our error handling and be updated to the correct range.

> If we change the name to targetUtilizationValue or targetUtilizationAmount, I think that there will be more confusion about what value or amount to put here.

I think I would rather be initially confused, than initially wrong.